manager/controllers/app/fybrik_notebook_test.go

```diff
@@ -110,55 +112,75 @@ func TestS3Notebook(t *testing.T) {
-	//nolint:govet
 	defer func() {
-		application := &apiv1alpha1.FybrikApplication{ObjectMeta: metav1.ObjectMeta{Namespace: applicationKey.Namespace,
-			Name: applicationKey.Name}}
-		_ = k8sClient.Get(context.Background(), applicationKey, application)
+		application := &apiv1alpha1.FybrikApplication{ObjectMeta: metav1.ObjectMeta{Namespace: writeApplicationKey.Namespace,
+			Name: writeApplicationKey.Name}}
+		_ = k8sClient.Get(context.Background(), writeApplicationKey, application)
 		_ = k8sClient.Delete(context.Background(), application)
 	}()
 
-	By("Expecting application to be created")
+	By("Expecting write application to be created")
 	g.Eventually(func() error {
-		return k8sClient.Get(context.Background(), applicationKey, application)
+		return k8sClient.Get(context.Background(), writeApplicationKey, writeApplication)
 	}, timeout, interval).Should(gomega.Succeed())
 	By("Expecting plotter to be constructed")
 	g.Eventually(func() *apiv1alpha1.ResourceReference {
-		_ = k8sClient.Get(context.Background(), applicationKey, application)
-		return application.Status.Generated
+		_ = k8sClient.Get(context.Background(), writeApplicationKey, writeApplication)
+		return writeApplication.Status.Generated
 	}, timeout, interval).ShouldNot(gomega.BeNil())
 
 	// The plotter has to be created
 	plotter := &apiv1alpha1.Plotter{}
-	plotterObjectKey := client.ObjectKey{Namespace: application.Status.Generated.Namespace, Name: application.Status.Generated.Name}
+	plotterObjectKey := client.ObjectKey{Namespace: writeApplication.Status.Generated.Namespace, Name: writeApplication.Status.Generated.Name}
 	By("Expecting plotter to be fetchable")
 	g.Eventually(func() error {
 		return k8sClient.Get(context.Background(), plotterObjectKey, plotter)
 	}, timeout, interval).Should(gomega.Succeed())
 
-	By("Expecting application to be ready")
+	By("Expecting write application to be ready")
 	g.Eventually(func() bool {
-		err = k8sClient.Get(context.Background(), applicationKey, application)
+		err = k8sClient.Get(context.Background(), writeApplicationKey, writeApplication)
 		if err != nil {
 			return false
 		}
-		return application.Status.Ready
+		return writeApplication.Status.Ready
 	}, timeout, interval).Should(gomega.Equal(true))
 
 	modulesNamespace := plotter.Spec.ModulesNamespace
 	fmt.Printf("data access module namespace notebook test: %s\n", modulesNamespace)
 
+	// Get the new connection details
+	var newBucket, newObject string
+	connectionMap := writeApplication.Status.ProvisionedStorage["new-data-parquet"].
+		Details.Connection.AdditionalProperties.Items
+	g.Expect(connectionMap).To(gomega.HaveKey("s3"))
+	s3Conn := connectionMap["s3"].(map[string]interface{})
+	g.Expect(s3Conn["endpoint"]).To(gomega.Equal("http://s3.fybrik-system.svc.cluster.local:9090"))
+	newBucket = fmt.Sprint(s3Conn["bucket"])
+	newObject = fmt.Sprint(s3Conn["object_key"])
+	g.Expect(newBucket).NotTo(gomega.Equal(""))
+	g.Expect(newObject).NotTo(gomega.Equal(""))
+
```

Comment:
Check that the returned assetID is not empty.

Reply:
Done. Thanks